src/main/java/uk/gov/justice/hmpps/prison/api/model/RequestForNewBooking.java

```diff
@@ -31,7 +31,7 @@ public class RequestForNewBooking {
     @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
     private LocalDateTime bookingInTime;
 
-    @Schema(description = "Where the prisoner has moved from (default OUT)", example = "OUT")
+    @Schema(description = "Where the prisoner has moved from (default OUT)", example = "SHEFCC")
     @Length(max = 6, message = "From location")
```

Comment:
turns out you can't used "OUT" but if you don't supply a location it will default to "OUT"  (TBH it is probably a bug)